ShoppingCart/Product.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace ShoppingCart
+{
+    public class Product
+    {
+        public string Name { get;  }
+        private Category _category;
+        public double Price { get;  }
+        public Category Category
```

Comment:
Why special treatment for category?? you can declare as you declared Name and Price

Reply:
I have done the changes suggested

Reply:
Kumar,

Please can you explain what fixed and configurable discount is from the consumer's point of view?